recipes/compaction/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.1.2" %}
+
+package:
+  name: compaction
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/c/compaction/compaction-{{ version }}.tar.gz
+  sha256: f13dfad27140295f850a1ea8a8b140d5ce483b3d3a4f2ca701203b7a9493ee87
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
It can be just
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

Reply:
Thanks @marcelotrevisani. I've made this change.